pages/callbacks/overview_callbacks.py

```diff
@@ -224,89 +226,78 @@ def total_contributor_growth(data, bin_size):
 
     """
-        Group-by determined by the radio button options.
-        Aggregation is the number of rows per time bin.
-        Days, Months, Years are all options.
+    Group-by determined by the radio button options.
+    Aggregation is the number of rows per time bin.
+    Days, Months, Years are all options.
     """
-    if(bin_size == "D1"):
+    if bin_size == "D1":
         group = df_contrib.groupby(
-            pd.Grouper(key='created_at', axis=0, 
-                      freq='1D')).size()
-    elif(bin_size == "M1"):
+            pd.Grouper(key="created_at", axis=0, freq="1D")
+        ).size()
+    elif bin_size == "M1":
         group = df_contrib.groupby(
-            pd.Grouper(key='created_at', axis=0, 
-                      freq='1M')).size()
+            pd.Grouper(key="created_at", axis=0, freq="1M")
+        ).size()
     else:
         group = df_contrib.groupby(
-            pd.Grouper(key='created_at', axis=0, 
-                      freq='1Y')).size()
+            pd.Grouper(key="created_at", axis=0, freq="1Y")
+        ).size()
 
     # reset index from group-by aggregation step
     group = group.reset_index()
     # rename the columns for clarity
-    group = group.rename(columns={'created_at': 'date', 0:'count'})
+    group = group.rename(columns={"created_at": "date", 0: "count"})
 
-    # correction for year binning - 
+    # correction for year binning -
     # rounded up to next year so this is a simple patch
-    if(bin_size == "M12"):
+    if bin_size == "M12":
         group["date"] = group["date"].dt.year
 
     # create the graph
-    fig = px.bar(group, x="date", y='count')
+    fig = px.bar(group, x="date", y="count")
 
     # make the bars thicker for further-spaced values
     # so that we can see the per-day increases clearly.
-    fig.update_traces(marker_color='blue',
-                  marker_line_color='blue',
-                  selector=dict(type="bar"))
+    fig.update_traces(
+        marker_color="blue", marker_line_color="blue", selector=dict(type="bar")
+    )
 
     # add the date-range selector
     fig.update_layout(
         xaxis=dict(
             rangeselector=dict(
-                buttons=list([
-                    dict(count=1,
-                        label="1m",
-                        step="month",
-                        stepmode="backward"),
-                    dict(count=6,
-                        label="6m",
-                        step="month",
-                        stepmode="backward"),
-                    dict(count=1,
-                        label="YTD",
-                        step="year",
-                        stepmode="todate"),
-                    dict(count=1,
-                        label="1y",
-                        step="year",
-                        stepmode="backward"),
-                    dict(step="all")
-                ])
-            ),
-            rangeslider=dict(
-                visible=True
+                buttons=list(
```

Comment:
Is this change against our set formatting of breaking into lines?

Reply:
It's because of the Black formatter that Michael Clifford suggested.